internal/cypress/docker/docker.go

```diff
@@ -206,9 +206,12 @@ func (handler *Handler) StartContainer(ctx context.Context, c cypress.Project) (
 		return nil, err
 	}
 
-	m, err := createMounts(files, pDir)
-	if err != nil {
-		return nil, err
+	var m []mount.Mount
+	if c.Docker.FileTransfer == config.DockerFileMount || c.Docker.FileTransfer == "" {
```

Comment:
That works! Though may I suggest defaulting the variable in the place where the config is being read (i.e. https://github.com/saucelabs/saucectl/blob/master/internal/cypress/config.go#L49)?

There we could do something like this (written in prose):
```
if FileTransfer == "" {
 FileTransfer = config.DockerFileMount
}
```

That way we also know later on that the setting has very explicit values and don't have to check for the empty-case any longer :)